addons/mail/tests/test_res_partner.py

```diff
@@ -591,3 +592,13 @@ def test_partner_merge_wizards(self):
         all_msg = p2_msg_ids_init + p1_msg_ids_init + p1_msg1
         self.assertEqual(len(p2.message_ids), len(all_msg) + 1, 'Should have original messages + a log')
         self.assertTrue(all(msg in p2.message_ids for msg in all_msg))
+
+    def test_partner_share(self):
+        """ Test partners with company e-mail domain are not considered customers.
+        Otherwise base_automation is treating outgoing mails as incoming for example. """
+        alias_domain = self.env.company._default_alias_domain_id()
+        alias_domain.name = 'example.org'
+        self.env.company.partner_id.email = 'info@example.org'
+        self.env.flush_all()
+        self.cr.precommit.run()
+        self.assertFalse(self.env.company.partner_id.partner_share)
```

Comment:
Can you add a test case for your issue instead, in base_automation-related tests?